src/main/java/io/minepkg/testutils/gui/WSpriteButton.java

```diff
@@ -33,14 +35,15 @@ public void paint(MatrixStack matrices, int x, int y, int mouseX, int mouseY) {
     if (!isEnabled()) state = 0;
     else if (hovered) state = 2;
 
-    ScreenDrawing.coloredRect(x, y, getWidth(), getHeight(), 0xFF_7b7b7b);
-    ScreenDrawing.coloredRect(x+1, y+1, getWidth()-2, getHeight()-2, state == 2 ? 0xFF_ababab : 0xFF_b7b7b7);
+    RenderSystem.setShaderColor(1.0f, 1.0f, 1.0f, 1.0f);
```

Comment:
This is necessary to ensure `coloredRect` uses the correct colors. Without it, it looked like
![bad_colors](https://user-images.githubusercontent.com/8464472/125672801-624c5557-7317-42bf-af21-45361b45ea41.png)
The reason why the 2nd and 3rd buttons turned out okay is because the `texturedRect()` below (re)sets the colors.